.github/workflows/typos.yml

```
@@ -0,0 +1,17 @@
+name: Test GitHub Action
+on: [push, pull_request]
+
+jobs:
+  run:
+    name: Spell Check with Typos
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Actions Repository
+      uses: actions/checkout@v4
+
+    - name: Writes changes in the local checkout
+      uses: crate-ci/typos@master
```

Please confirm that you meant to use the master branch

---

We can just use the latest release.
```suggestion
      uses: crate-ci/typos@v1.23.6
```